x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -62,6 +62,26 @@ export class CspPlugin
   private readonly logger: Logger;
   private isCloudEnabled?: boolean;
 
+  /**
+   * CSP is initialized when the Fleet package is installed.
+   * either directly after installation, or
+   * when the plugin is started and a package is present.
+   */
+  readonly #initializerTracker = (() => {
```

Comment:
nit I'm wondering why is it not a simple data member of this class. The owner of this state is purely `CspPlugin` and shouldn't be altered from outside of it

someone can turn this off although it is initialized

or in other words making `initialized` with public getter and private setter makes more sense here to me (`on` and `off` are public setters if I understand correctly)

Reply:
changed to a boolean property on `plugin` to keep it simple. if you meant something else, please clarify. not quite sure what the description explains. (what is a data member?)